File: appserver-legacy/src/main/java/org/radarbase/appserver/dto/protocol/NotificationProtocolMode.java
package org.radarbase.appserver.dto.protocol;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum NotificationProtocolMode {
    @JsonProperty("standard")
    STANDARD,
    @JsonProperty("disabled")
    DISABLED,
    @JsonProperty("combined")
    COMBINED,
}

File: appserver-jersey/src/main/kotlin/org/radarbase/appserver/jersey/dto/protocol/NotificationProtocolMode.kt
package org.radarbase.appserver.jersey.dto.protocol

import com.fasterxml.jackson.annotation.JsonProperty

enum class NotificationProtocolMode {
    @JsonProperty("standard")
    STANDARD,

    @JsonProperty("disabled")
    DISABLED,

    @JsonProperty("combined")
    COMBINED,
}
